plugins/modules/packaging/language/gem.py

```diff
@@ -62,6 +62,17 @@
       These gems will be independent from the global installed ones.
       Specifying this requires user_install to be false.
     required: false
+  bindir:
+    type: path
+    description:
+    - Install executables into a specific directory.
+    version_added: 3.3.0
+  norc:
+    type: bool
+    default: false
+    description:
+    - Avoid loading any C(.gemrc) file. Ignored for RubyGems prior to 2.5.2.
```

Comment:
Please note deprecation and change notice in c.g. versions here.

Reply:
I would appreciate help in phrasing it here.

Here is what @felixfontein suggested:
1.  Add a new option which enables this. Default value is to disable.
2. In community.general 4, remove the default, and print out a deprecation message that the default will change in community.general 5.0.0 to the new value.
3. In community.general 5.0.0, switch the default.